tools/report_generators/reporters.py

```diff
@@ -189,7 +189,7 @@ def __init__(self, region, wwwroot="wwwroot", verbose=False):
             overview_function="overview",
             overview_args=f'country="Germany", subregion="{self.subregion}"',
             compare_plot_function="compare_plot",
-            compare_plot_args=f'country="Germany", subregion="{self.subregion}"',
+            compare_plot_args=f'country="Germany", subregion="{self.subregion}", dates="2020-03-15:"',
```

Comment:
Why do we restrict the data to be after 15 March 2020 here?

Reply:
As I wrote above, this date was taken rather arbitrarily. My idea was that it's a good approximation of the beginning of the first wave in Germany (see the picture below). We can take any other date we find more suitable, there's no objections from my side.

![image](https://user-images.githubusercontent.com/1487169/98463077-0dda2a80-21b9-11eb-9739-9b6e3542ea72.png)